src/lib/viewers/controls/settings/SettingsAutoplay.tsx

```diff
@@ -1,34 +1,34 @@
 import React from 'react';
-import MediaSettings, { Menu } from '.';
+import Settings, { Menu } from '.';
 
 export type Props = {
     autoplay: boolean;
     onAutoplayChange: (autoplay: boolean) => void;
 };
 
-export default function MediaSettingsMenuAutoplay({ autoplay, onAutoplayChange }: Props): JSX.Element {
-    const { setActiveMenu } = React.useContext(MediaSettings.Context);
+export default function SettingsMenuAutoplay({ autoplay, onAutoplayChange }: Props): JSX.Element {
```

Comment:
Let's leave the consumers of the base components in the `media` folder, since they're specific to those viewers.